.github/workflows/ci.yml

```
@@ -114,10 +114,11 @@ jobs:
         with:
           images: gresearchdev/siembol-${{ matrix.component }}
           flavor: |
-            latest=${{ github.ref == 'refs/heads/master' }}
+            latest=${{ github.ref == 'refs/heads/master' && !endsWith(steps.info.outputs.version, 'SNAPSHOT') }}
```

I think we should add the hyphen to the suffix check. Could you please apply the same suggestion for all `latest=` lines?
```suggestion
            latest=${{ github.ref == 'refs/heads/master' && !endsWith(steps.info.outputs.version, '-SNAPSHOT') }}
```

---

added hyphen to condition + if you agree I've added also for `-dev` condition which should match current format `"version": "1.3.2-dev"`